test/images/image-util.sh

```diff
@@ -262,6 +262,15 @@ if [[ "${WHAT}" == "all-conformance" ]]; then
   for image in "${conformance_images[@]}"; do
     eval "${TASK}" "${image}" "$@"
   done
+elif [[ "${WHAT}" == "updated-images" ]]; then
+  # if updated-images is selected it obtains the images modified respect HEAD and updates them
+  # this is required for the postsubmit job used to update the statging repo and not having to
+  # rebuild all the images
+  shift
+  updated_images=$(git diff --dirstat=files,0 HEAD~1 -- ${KUBE_ROOT}/test/images/* | cut -d "/" -f 3)
```

Comment:
@BenTheElder you are the expert here, how can I get the diff  in the PR? against master?